src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -65,56 +80,78 @@
  */
 public class DynamicACLFilter 
 	extends RestFilter 
-	implements ConfigurationSettings, SearchGuardACLActionRequestListener {
+	implements ConfigurationSettings {
 
 	private static final String AUTHORIZATION_HEADER = "Authorization";
-	private static final String SEARCHGUARD_TYPE = "ac";
-	private static final String SEARCHGUARD_ID = "ac";
-	private static final String[] BLACKLIST_PROJECTS = {"default", "openshift", "openshift-infra" };
+	private static final String[] BLACKLIST_PROJECTS = {"default", "openshift", "openshift-infra", "kube-system" };
```

Comment:
Should this be configurable?  I dont recall if these are just the defaults, but we may have need to add to this list.